src/viur/core/bones/relational.py

```diff
@@ -600,7 +592,7 @@ def postSavedHandler(self, skel, boneName, key):
                 dbObj["viur_delayed_update_tag"] = time()
                 dbObj["viur_relational_updateLevel"] = self.updateLevel.value
                 dbObj["viur_relational_consistency"] = self.consistency.value
-                dbObj["viur_foreign_keys"] = self.refKeys
+                dbObj["viur_foreign_keys"] = tuple(self.refKeys)
```

Comment:
Due https://github.com/viur-framework/viur-datastore/issues/46, we need to use `list()` here.
```suggestion
                dbObj["viur_foreign_keys"] = list(self.refKeys)
```